_events/sr2019/competition.md

```diff
@@ -3,24 +3,50 @@ title: Competition
 date: 2019-04-06 9:00:00
 layout: event
 type: competition
-location: Southampton University Highfield Campus
+location: Southampton University
 ---
 
 The Student Robotics Competition marks the ultimate stage of SR2019, showcasing the months of preparation. Our teams will work hard to push their robots to the limit and quickly make repairs between league matches. We will finish the weekend with the finals where the best bots will be named victorious.
 
-## Directions
+## Schedule
 
-The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. 
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The exact start/end times are to be confirmed, but doors open 9am and we aim to finish around 6pm.
 
-* How to [get to campus][soton-campus-directions]
+The Saturday will begin with a briefing on the rules and general struture of the weekend, followed by some time for teams to test their robots in the arena. Soon after, the league matches will start, in which your robot will compete against 3 other bots to rise up the league. These continue for the rest of the day.
 
-## Schedule
+On Sunday morning, the league matches will continue. Once these have concluded, we will begin knockout matches, the order of which is determined by your position in the league. The reamining teams will compete in the finals, followed by the awards for the winning teams.
 
-The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The exact start/end times are to be confirmed, but doors open 9am and we aim to finish around 6pm.
+## Tickets and Media Consent
+
+Competitors need to bring along a ticket to be allowed entry. Anyone under the
+age of 18 must be accompanied by an adult. Student Robotics will announce when
+tickets are available.
+
+All competitors and visitors will need to complete a media consent form and bring this along too.
```

Comment:
Is this how we're going to do this? In the past we've had them submit the media consent forms ahead of time, for conversion into a ticket (this is the entire point of having tickets -- that they're easier to process than media consent forms).

Reply:
Correct, media consent will need to be sent to us before the event. We should encourage people to send then in beforehand

Reply:
https://github.com/srobo/website/pull/132/commits/960d3d4d6da64da341c148fbaba151747e2df1e2